include/netutils/webclient.h

```diff
@@ -147,6 +281,12 @@ int wget(FAR const char *url, FAR char *buffer, int buflen,
 int wget_post(FAR const char *url, FAR const char *posts, FAR char *buffer,
               int buflen, wget_callback_t callback, FAR void *arg);
 
+void webclient_set_defaults(FAR struct webclient_context *ctx);
```

Comment:
Should we unify the prefix? either wget_xxx or webclient_xxx, but not both.

Reply:
i plan to remove all wget_ things after adapting users.

Reply:
ping @yamt any news?

Reply:
@acassis 
no news. are you waiting for this PR "complete"?
like adapting users, removing the old api, etc, in a single shot?

i prefer to work on a step-by-step way.
(as a bonus, it might give external users time to adapt their code to the new api.)
but if you prefer a single-shot, i can try.

Reply:
@yamt please do it step by step

Reply:
if you prefer step-by-step, this PR (the first step) is ready to review/merge already.